anvil-rstudio-bioconductor/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.11 - 5/4/2021
+
+- Update in base image `biconductor/bioconductor_docker:RELEASE_3_12`. 
```

Comment:
was there no version bump of the base image for the bug fix?

Reply:
We mark bug fixes internally inside the Dockerfile, https://github.com/Bioconductor/bioconductor_docker/blob/RELEASE_3_12/Dockerfile#L10